ezgui/src/event_ctx.rs

```diff
@@ -184,7 +184,7 @@ impl<'a> LoadingScreen<'a> {
             &draw,
         );
 
-        g.inner.finish();
+        g.prerender.inner.draw_finished(g.inner);
```

Comment:
This change was the fallout from making it so that GfxCtx didn't have to know about window vs windowed_context